apps/home/routes.coffee

```diff
@@ -25,6 +25,14 @@ positionWelcomeHeroMethod = (req, res) ->
   res.cookie 'hide-welcome-hero', '1', expires: new Date(Date.now() + 31536000000)
   method
 
+maybeFetchHomepageRails = (req)->
+  deferred = Q.defer()
+  metaphysics(query: query, req: req)
+    .then (data) -> deferred.resolve data
+    .catch -> deferred.resolve home_page: { artwork_modules: [] }
```

Comment:
So to be clear, the `metaphysics` function in Force rejects a promise when the GraphQL JSON response contains an `errors` entry? Or is this about connection failures between Force and metaphysics?